pages/views/elements.py

```diff
@@ -218,3 +226,44 @@ def get_context_data(self, **kwargs):
                         kwargs=url_kwargs),
                 })
         return context
+
+
+class CertificateDownloadView(LoginRequiredMixin, DetailView):
```

Comment:
I am not sure we want to have a `LoginRequiredMixin` here. The djaodjin-pages app is used within other projects that handle role-based access control (RBAC) through URL decorators.